tests/src/test/java/alluxio/master/journal/options/JournalReaderOptionsTest.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.journal.options;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link JournalReaderOptions}.
+ */
+public final class JournalReaderOptionsTest {
+  @Test
+  public void defaults() {
+    JournalReaderOptions options = JournalReaderOptions.defaults();
+    Assert.assertEquals(0, options.getNextSequenceNumber());
+    Assert.assertEquals(false, options.isPrimary());
+  }
+
+  /**
+   * Tests getting and setting fields.
+   */
+  @Test
+  public void fields() {
+    JournalReaderOptions options = JournalReaderOptions.defaults();
+    options.setPrimary(true);
```

Comment:
nit: I prefer to generate random values for the fields to improve test coverage across different iterations of the test:

```
Random random = new Random()
boolean primary = random.nextBoolean();
long nextSequenceNumber = random.nextLong();
options.setPrimary(primary);
options.setNextSequenceNumber(nextSequenceNumber);
Assert.assertEquals(primary, options.isPrimary());
Assert.assertEquals(nextSequenceNumber, options.getNextSequenceNumber());
```